docs/use-keyring-file.md

```diff
@@ -0,0 +1,57 @@
+# Use the keyring file component or keyring file plugin
+
+## Use the keyring file plugin
+
+Percona Server for MySQL supports the keyring_file plugin that stores the keyring data locally.
+
+!!! warning
+
+    The `keyring_file` plugin should not be used for regulatory compliance.
+
+To install the plugin, follow the [installing and uninstalling plugins] instructions.
+
+### Load the keyring plugin
+
+You should load the plugin at server startup with the `--early-plugin-load` option to enable keyrings.
+
+We recommend that you load the plugin in the configuration file to facilitate recovery for encrypted tables. Also, the redo log encryption and the undo log encryption cannot be used without `--early-plugin-load`. The normal plugin load happens too late at startup.
+
+To load the keyring_file plugin, run the following command:
+
+```{.bash data-prompt="$"}
+$ mysqld --early-plugin-load="keyring_file=keyring_file.so"
+```
+
+!!! note
```

Comment:
This doesn't need to be a note - just text.

Reply:
fixed